docs/agents/sematext-agent/kubernetes/helm.md

```diff
@@ -0,0 +1,108 @@
+## Introduction
+
+This chart installs the Sematext Agent to all nodes in your cluster via a `DaemonSet` resource.
+
+## Prerequisites
+
+- Kubernetes 1.13+
+- You need to create [an Infra App in Sematext Cloud](https://apps.sematext.com/ui/monitoring-create/app/infra) to get your infra token.
+
+## Installation
```

Comment:
Since all our examples use --region="US" maybe here in the intro we can say something about using EU if one uses SC EU.  Aha, I see that at the bottom.  Since we read top to bottom, maybe moving to top would be more helpful to folks?

Reply:
I moved it to below the first example because above it seemed to break the flow of the text a bit. Wdyt?